### PR TITLE
Poster update

### PR DESCRIPTION
**Pull request by SamCroswell**
- Adds two kinds of posters: 'Legit' and 'Contraband'
- Legit posters have been added to the map in major hallways and various public areas, and contraband posters have been added to maintenance shafts and various nefarious areas.
- Posters now use pixel_x and pixel_y for placement on walls, this solves an issue where you could see the poster placed on the wall at any angle because the poster was put onto the turf that the wall occupied.
- Adds a 'Corporate Posters' crate, containing 5 legit posters.
- Removes the contraband poster from the art crate, replaces it with a legit poster.
- Adds a new helper proc for determining in which direction an object adjacent to another object is relative to the first object.
- Consolidates all posters into one file, as opposed to the separate files for Bay and /TG/ posters.
- Removed some of the lower-quality posters.

Examples of legit posters:
ertposter
reportcrime
pdaposter
legit

Examples of contraband posters:
revolver
esword
hackguide
contraband

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
:+1: 

Another thing I wanted to port, but never got around to.

**Comment by ZomgPonies** [comment]
Awesome yeah I was looking into that.

File: paradise.dme [hunk 1]
```diff
@@ -5,6 +5,79 @@
...
+#define FILE_DIR "sound/music"
+#define FILE_DIR "sound/piano"
+#define FILE_DIR "sound/turntable"
+#define FILE_DIR "sound/violin"
+#define FILE_DIR "sound/voice"
+#define FILE_DIR "sound/voice/complionator"
+#define FILE_DIR "sound/voice/Serithi"
+#define FILE_DIR "sound/vox"
+#define FILE_DIR "sound/vox_fem"
+#define FILE_DIR "sound/weapons"
```
Comment (Fox-McCloud) [inline paradise.dme]: Build->preferences->uncheck "automatically set FILE_DIR".

**Comment by fj45** [comment]
Were you able to port the new bay ones?


What posters did you remove?

**Pull request merged**